### ISSUE TITLE
Question prepare our own multi 2D pathological datasets for training the MultiTalent? 

### ISSUE THREAD

**Issue by Younger330** [MIC-DKFZ/MultiTalent#12]
Your work is greatly appreciated, and I a question about preparing our 2D pathological datasets for MultiTalent.

I've observed that your work is based on nnUnet, which is designed for 3D images. Consequently, I have developed a Unet architecture specifically tailored for training our 2D pathological datasets. Following the configuration outlined in the paper, I have assigned a separate channel for each class in the dataset, resulting in a total of 32 channels, and I have replaced softmax with sigmoid.

Now, I have a question regarding the training process. Should I expand the original ground truth channel of the dataset to 32 channels for training, or should I reduce the predictions from the 32-channel output to match the original size of the dataset?

**Comment by constantinulrich** [comment]
Hi, i am not sure if i understood your question correctly: 
Lets say we have 2 datasets with each 2 and 3 classes. Then, I trained the multitalent networks with 5 output channels  (2 for dataset 1 and 3 for dataset 2). Due to the Sigmoid function, no explicit background class is needed. 
The network then always predicts all classes, but the loss was only calculated for the classes of the corresponding dataset. 

Did this help?

Best,
Constantin

**Comment by Younger330** [comment]
> Hi, i am not sure if i understood your question correctly: Lets say we have 2 datasets with each 2 and 3 classes. Then, I trained the multitalent networks with 5 output channels (2 for dataset 1 and 3 for dataset 2). Due to the Sigmoid function, no explicit background class is needed. The network then always predicts all classes, but the loss was only calculated for the classes of the corresponding dataset.
> 
> Did this help?
> 
> Best, Constantin

Yes, your understanding is completely correct. Thank you for your help.
